modules/asg/main.tf

```diff
@@ -20,6 +20,13 @@ resource "aws_security_group" "asg_sg" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
That's too broad. I'd rather limit to CIDR range of vpc

Reply:
Resolved